app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,81 @@
+class TweetsController < ApplicationController
+
+  #### TWEETS INDEX ACTION ####
+
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      @user = User.find(session[:user_id])
+      erb :'tweets/tweets'
+    else
+      redirect '/login'
+    end
+  end
+
+  #### CREATE ACTIONS ####
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/new'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do 
+    if params[:content] == ""
+      redirect '/tweets/new'
+    else
+      user = current_user
+      @tweet = Tweet.create(content: params[:content], user_id: user.id)
```

Comment:
1. @tweet does not need to be an instance variable 
2. we already have current_user so no need for user variable 
3. We should build tweets using the associative create method. 
```ruby 
tweet = current_user.tweets.create(content: params[:content])
```